integration_test.go

```diff
@@ -141,15 +146,15 @@ func TestHTTPTriger(t *testing.T) {
 
 // TestBuildExamples ensures that the tinygo examples will build successfully.
 func TestBuildExamples(t *testing.T) {
-	for _, example := range []string{
-		"examples/http-tinygo",
-		"examples/http-tinygo-outbound-http",
-		"examples/tinygo-outbound-redis",
-		"examples/tinygo-redis",
-		"examples/tinygo-key-value",
-		"examples/variables-tinygo",
-	} {
-		build(t, example)
+	examples, err := os.ReadDir("examples")
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, example := range examples {
```

Comment:
Might actually need to 

example := example

Below to ensure this is captured in the parallel execution, but otherwise my LGTM still stands